test/integration/LoginEndpoints.test.js

```diff
@@ -0,0 +1,127 @@
+import 'babel-polyfill' // Needed because of mocha
```

Comment:
Actually we might not need this import anymore (we're not even using `mocha` anymore). Could you check if everything works without it? If yes, please remove it from other integration test files too! 👍